File: openrest4j-api/src/com/googlecode/openrest/LatLng.java
package com.googlecode.openrest;

import java.io.Serializable;

import org.codehaus.jackson.annotate.JsonIgnoreProperties;
import org.codehaus.jackson.map.annotate.JsonSerialize;

@JsonIgnoreProperties(ignoreUnknown = true)
public class LatLng implements Serializable {
    public LatLng(Double lat, Double lng) {
        this.lat = lat;
        this.lng = lng;
    }

    /** Default constructor for JSON deserialization. */
    public LatLng() {}

    @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
    public Double lat;

    @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
    public Double lng;
    
    private static final long serialVersionUID = 1L;
}

File: trunk/openrest4net/openrest4net-api/com/googlecode/openrest/LatLng.cs
using System;

namespace com.googlecode.openrest
{
    public class LatLng
    {
        public LatLng(double lat, double lng)
        {
            this.lat = lat;
            this.lng = lng;
        }

        /** Empty constructor required for initialization from JSON-encoded string. */
        public LatLng() { }

        public double? lat;
        public double? lng;
    }
}
